### PR TITLE
Add likert question type.

### PR DESCRIPTION
**Pull request by relumesaros**

### CODE REVIEW

File: packages/likert/package.json [hunk 1]
```diff
@@ -9,14 +9,17 @@
   "module": "src/index.js",
   "dependencies": {
     "@material-ui/core": "^3.9.2",
-    "@pie-framework/pie-player-events": "^0.1.0",
-    "@pie-lib/graphing": "^2.0.0",
+    "@pie-lib/config-ui": "^10.8.25",
```
Comment (edeustace) [inline packages/likert/package.json:12]: you shouldn't depend on config-ui in a ui package - can you remove this?

File: packages/likert/src/index.js [hunk 2]
```diff
@@ -1,58 +1,97 @@
...
+        } else {
+          log('skip');
+        }
+      },
+      50,
+      { leading: false, trailing: true }
+    );
+
+    this._dispatchResponseChanged = debounce(() => {
+      const event = new CustomEvent('session-changed', {
```
Comment (edeustace) [inline packages/likert/src/index.js:40]: can you use the events from this package instead: https://github.com/pie-framework/pie-player-events

File: packages/likert/src/index.js [hunk 3]
```diff
@@ -1,58 +1,97 @@
...
+        }
+      });
+
+      this.dispatchEvent(event);
+    });
+
+    this._dispatchModelSet = debounce(
+      () => {
+        this.dispatchEvent(
+          new CustomEvent('model-set', {
```
Comment (edeustace) [inline packages/likert/src/index.js:55]: use the events from this package: https://github.com/pie-framework/pie-player-events

File: packages/likert/package.json (hunk 1 above)
Comment (relumesaros) [reply packages/likert/package.json]: solved

File: packages/likert/src/index.js (hunk 2 above)
Comment (relumesaros) [reply packages/likert/src/index.js]: solved

File: packages/likert/src/index.js (hunk 3 above)
Comment (relumesaros) [reply packages/likert/src/index.js]: solved

**Comment by edeustace** [comment]
:pray: @relumesaros

**Pull request merged**